tests/models/whisper/test_modeling_whisper.py

```diff
@@ -1653,7 +1653,11 @@ def test_large_logits_librispeech(self):
 
         processor = WhisperProcessor.from_pretrained("openai/whisper-large")
         processed_inputs = processor(
-            audio=input_speech, text="This part of the speech", add_special_tokens=False, return_tensors="pt"
+            audio=input_speech,
+            text="This part of the speech",
+            add_special_tokens=False,
+            return_tensors="pt",
+            sampling_rate=16_000,
```

Comment:
Adding the arg `sampling_rate=16_000` whenever we call the processor significantly reduces the number of warnings on the logger